test/e2e/tls/client-ca/server.crt

```diff
@@ -0,0 +1,16 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Self signed certs are good for tests, can you tell me when they expire? Sorry cant decode them at the moment

Reply:
Both server and client certs are valid in 10 years. 2030.

![Screen Shot 2020-08-25 at 11 41 06 AM](https://user-images.githubusercontent.com/34478078/91196304-f7e3c000-e6c7-11ea-921e-5eeb5cec62bf.png)
![Screen Shot 2020-08-25 at 11 42 27 AM](https://user-images.githubusercontent.com/34478078/91196392-11850780-e6c8-11ea-885d-06791d3fbc81.png)

Reply:
👍